Students/tylerkobienia/PART3LAB2/PART3LAB2.ino

```diff
@@ -0,0 +1,38 @@
+
+#define RED 13
+#define BLUE  12
+#define Temp_Sensor A0
+#define MAXTEMP 75 // define the threshold temperature
+#define MIN_TEMP 32 // define the range of temperature values
+#define MAX_TEMP 212
```

Comment:
FYI, having `MAXTEMP` and `MAX_TEMP` could be very confusing for reading the code.